ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -231,97 +231,114 @@ Item {
             }
         }
 
-        StatusChatInput {
-            id: chatInput
-
+        RowLayout {
             Layout.fillWidth: true
             Layout.margins: Style.current.smallPadding
 
-            enabled: root.rootStore.sectionDetails.joined && !root.rootStore.sectionDetails.amIBanned &&
-                     root.rootStore.isUserAllowedToSendMessage
+            StatusChatInput {
```

Comment:
I swear I only moved this into a `RowLayout`. 
The contents in `StatusChatInput` remain the same (except removing `isContactBlocked` property)